### PR TITLE
Swift: add consistency check and accept results for the moment

### PR DESCRIPTION
**Pull request by redsun82**

### CODE REVIEW

**Comment by redsun82** [comment]
@geoffw0 do we need a change note for this, because of the newly added `consistency-queries` pack?

**Comment by geoffw0** [comment]
It's probably a good idea because users may notice this, yeah.  If we don't expect users to want to use the consistency pack, the change note can be very brief.

File: swift/ql/test/query-tests/Security/CWE-916/CONSISTENCY/CfgConsistency.expected [hunk 1]
```diff
@@ -0,0 +1,5 @@
+deadEnd
+| file://:0:0:0:0 |  ... = ... |
+| file://:0:0:0:0 |  ... = ... |
+| file://:0:0:0:0 |  ... = ... |
+| file://:0:0:0:0 |  ... = ... |
```
Comment (geoffw0) [inline swift/ql/test/query-tests/Security/CWE-916/CONSISTENCY/CfgConsistency.expected:5]: How do these tests work?  I can only see the `.expected` files in each test directory.

Comment (geoffw0) [reply swift/ql/test/query-tests/Security/CWE-916/CONSISTENCY/CfgConsistency.expected:5]: Ah I see, thanks.

**Review by geoffw0 — APPROVED** [review]
It will be good to have the consistency checks running more widely.  Intuitively I'm not sure how valuable running them on every single _query_ test is going to be in the long run ... but then again, right now they're finding stuff all over.

LGTM.

**Pull request merged**